### PR TITLE
Updated CSS to fix regression in top bar

### PR DESCRIPTION
**Pull request by m-gb**
I removed the display:block in the profile-header CSS class.
The avatar and user name are now aligned.

### CODE REVIEW

File: app/assets/stylesheets/custom/_layout.scss [hunk 1]
```diff
@@ -150,10 +150,7 @@ nav.navbar {
     align-self: center;
   }
   &.person-profile {
-    display: none;
-    @media (min-width: $mq-large) {
-      display: block;
```
Comment (lislis) [inline app/assets/stylesheets/custom/_layout.scss]: The easiest would be to just change this line to `display: flex`

**Comment by m-gb** [comment]
Thank you for responding, I didn't know that was the expected behavior.
I updated the pr, let me know if it's ok.

**Comment by til** [comment]
Great, thanks! Yes, this way is the right one, the navbar looks good now.

I reverted the change to the `.ruby-version` file and squashed the commits into one, which is the reason why your commit is now in master: https://github.com/rubycorns/rorganize.it/commit/09dfa2434ccc00e6af3d1e58fa50c1371c868047 even though I'm just closing this PR now without merging.

Also it's deployed live: https://rorganize.it/

**Comment by lislis** [comment]
yaaay

**Pull request closed without merging**